src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -51,6 +52,7 @@ export const formConfigs = {
   '22-1990E': edu1990eConfig,
   '22-1990N': edu1990nConfig,
   '22-1995': edu1995Config,
+  '22-1995-STEM': edu1995StemConfig,
```

Comment:
this should have been part of https://github.com/department-of-veterans-affairs/vets-website/pull/10043